package.json

```diff
@@ -103,6 +109,8 @@
     "happypack": "3.0.1",
     "html-webpack-plugin": "2.24.1",
     "json-loader": "0.5.4",
+    "koa-flow-declarations": "^0.1.0",
```

Comment:
Please use "exact" versions. i.e. `0.1.0` vs `~0.1.0`.  You'll probably need to kill and rebuild the yarn.lock file.  When using yarn to add dependencies (as is recommended) pass a `--exact` flag to make sure it puts exact version numbers in the package.json dependency lists.